packages/platforms/browser/lib/auto-instrumentation/network-request-plugin.ts

```diff
@@ -60,6 +60,7 @@ export class NetworkRequestPlugin implements Plugin<BrowserConfiguration> {
   }
 
   private shouldTrackRequest (startContext: RequestStartContext): boolean {
-    return startContext.url !== this.configEndpoint
+    const permittedPrefixes = ['http://', 'https://', '/', './', '../']
```

Comment:
This makes a new array every time `shouldTrackRequest` is called; can we define this in the top-level so we only make the array once?